.openshift-ci/jobs/integration-tests/envbuilder.sh

```diff
@@ -0,0 +1,289 @@
+            _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo reboot" || true
+            return 0
+        fi
+        echo "Retrying in 5s ..."
+        sleep 5
+    done
+    echo "Failed to install ESM updates after 3 retries"
+    return 1
+}
+
+installFIPSOnUbuntuAndReboot() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_SSH_KEY_FILE="$1"
+    shift
+    for _ in {1..3}; do
+        if _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo ua enable --assume-yes fips"; then
+            return 0
+        fi
+        echo "Retrying in 5s ..."
+        sleep 5
+    done
+    echo "Failed to install FIPS after 3 retries"
+    return 1
+}
+
+rebootVM() {
+    local GCP_VM_USER="$1"
+    shift
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_SSH_KEY_FILE="$1"
+    shift
+
+    # Restart the VM.
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo reboot" || true
+
+    sleep 5
+
+    # Ensure the VM is up and SSH is available.
+    gcpSSHReady "$GCP_VM_USER" "$GCP_VM_NAME" "$GCP_SSH_KEY_FILE"
+}
+
+installDockerOnRHELViaGCPSSH() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_IMAGE_FAMILY="$1"
+    shift
+    local GCP_SSH_KEY_FILE="$1"
+    shift
+
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo yum install -y yum-utils device-mapper-persistent-data lvm2"
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo"
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo yum-config-manager --setopt=\"docker-ce-stable.baseurl=https://download.docker.com/linux/centos/${GCP_IMAGE_FAMILY: -1}/x86_64/stable\" --save"
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo yum install -y docker-ce docker-ce-cli containerd.io"
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo systemctl start docker"
+}
+
+setupDockerOnSUSEViaGCPSSH() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_SSH_KEY_FILE="$1"
+    shift
+
+    _ gcloud compute ssh --ssh-key-file="${GCP_SSH_KEY_FILE}" "$GCP_VM_NAME" --command "sudo systemctl start docker"
+}
+
+gcpSSHReady() {
+    local GCP_VM_USER="$1"
+    shift
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_SSH_KEY_FILE="$1"
+    shift
+
+    local retryCount=6
+    for _ in $(seq 1 $retryCount); do
+        _ gcloud compute ssh --strict-host-key-checking=no --ssh-flag="-o PasswordAuthentication=no" --ssh-key-file="${GCP_SSH_KEY_FILE}" "${GCP_VM_USER}@${GCP_VM_NAME}" --command "whoami" \
+            && exitCode=0 && break || exitCode=$? && sleep 15
+    done
+
+    instance_id="$(_ gcloud compute instances describe "${GCP_VM_NAME}" --format='get(id)')"
```

Comment:
Remove the usage of `_` here. It echos the command and the `instance_id` var will be invalid.